.github/actions/setup-linux/action.yml

```diff
@@ -15,6 +15,7 @@ runs:
   using: "composite"
   steps:
     - name: Use ccache
+      if: ${{ runner.arch == 'x86_64' }}
```

Comment:
@awni This disables ccache completely as `x86_64` is not a valid value for `runner.arch`. Do you remember why it is disabled for arm build? Maybe we should add an input to just disable ccache for release builds?